src/Controller/DashboardController.php

```diff
@@ -59,27 +59,39 @@ public function ajaxTab(UserInterface $user, $subpage) {
       'subpage' => $subpage,
     ]);
 
-    $content = [];
+    $content = [
+      '#theme' => 'contacts_dash_tab_content',
+      '#content' => [
```

Comment:
There's got to be a better way than using `#content` for this, can left/right not just be children that get put in specific regions? I feel it makes the render array very messy...